src/views/SearchResults.vue

```diff
@@ -108,6 +108,11 @@ export default {
 			return store.state.searchData;
 		},
 	},
+	watch: {
+		searchDataFromVuex() {
+			//handle searching here, I am leaving this here becuase of how router works, you cannot use mounted
```

Comment:
What's this for?

Reply:
This is for future to use watcher for downloading data from API because of how router works.
It does not allow you to go to the same page twice, so for example if you want to put the function for downloading in mounted() it would work only once.

I can delete this if you think it's unneccessary.